packages/cyberstorm/README.md

```diff
@@ -0,0 +1,76 @@
+# Cyberstorm UI library
+
+## Storybook
+
+[Storybook](https://storybook.js.org/docs/react/get-started/introduction)
+provides a sandbox to build UI components in isolation, without having to start
+up the whole service stack. Additionally it showcases the existing components,
+promoting reusability.
+
+To start Cyberstorm Storybook, run `yarn workspace @thunderstore/cyberstorm-storybook storybook`.
+
+Storybook can then be accessed at [http://localhost:6006/].
+
+## Naming conventions
+
+### Basic information
+
+CSS ClassNames should follow camelCase format, e.g.:
+
+- `.root`
+- `.label`
+- `.categoryWrapper`
+
+Variation declarations should be separated from the class by two underscores, e.g.:
+
+- `.packageCard__default`
+- `.metaItem__last`
+- `.button__specialGreen`
+
+Component's root element should have the class name of `.root`
```

Comment:
Should this be mentioned between lines 22 and 24? Now the content kinda jumps CSS classes -> variations -> CSS classes -> Variations?

Reply:
Moved things around 👍